samples/gen-go-db/server/db/dynamodb/dynamodb_test.go

```diff
@@ -22,20 +21,24 @@ func TestDynamoDBStore(t *testing.T) {
 	testCtx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	cmd := exec.CommandContext(testCtx, "./dynamodb-local.sh")
-	ddbLocalOutputReader, ddbLocalOutputWriter := io.Pipe()
-	cmd.Stdout = io.MultiWriter(os.Stdout, ddbLocalOutputWriter)
-	cmd.Stderr = io.MultiWriter(os.Stderr, ddbLocalOutputWriter)
 	if err := cmd.Start(); err != nil {
 		t.Fatal(err)
 	}
 
-	// wait for dynamodb local to output the correct startup log
-	scanner := bufio.NewScanner(ddbLocalOutputReader)
-	for scanner.Scan() {
-		if strings.Contains(scanner.Text(), "Initializing DynamoDB Local with the following configuration") {
+	end := time.Now().Add(60 * time.Second)
+	var c net.Conn
```

Comment:
maybe move this into the for loop and close it in the for loop? I'm actually not entirely sure what happens in the case where you overwrite a connection like this...